pkg/types/chain_test.go

```diff
@@ -42,7 +42,7 @@ func TestGetLinksPresent(t *testing.T) {
 
 	assert.Len(t, links, 2, "Expected 2 links")
 	assert.Equal(t, "Keplr", links[0].Name, "Expected Keplr link")
-	assert.Equal(t, "https://wallet.keplr.app/#/chain/governance?detailId=test", links[0].Href, "Wrong Keplr link")
+	assert.Equal(t, "https://wallet.keplr.app/chains/chain/proposals=test", links[0].Href, "Wrong Keplr link")
```

Comment:
```suggestion
	assert.Equal(t, "https://wallet.keplr.app/chains/chain/proposals/test", links[0].Href, "Wrong Keplr link")
```
This one should work, I tested it locally. Can you add it to your branch? @hakuno2000

Reply:
Ok sir